stdlib.sh

```diff
@@ -389,6 +453,106 @@ function __STDLIB_oneshot_syntax_check() { # {{{
 
 ###############################################################################
 #
+# stdlib.sh - Initialise coloured output
+#
+###############################################################################
+
+function __STDLIB_oneshot_colours_init() { # {{{
+
+	if(( std_COLOUR_ON )); then
+		local foregroundInfo="white"
+		local backgroundInfo=""
+		local modeInfo=""
```

Comment:
Sorry, I still found it quite effective when reading code.
And I hate underscores! :P